docs/docs/aws.md

```diff
@@ -565,6 +565,29 @@ spark-sql --conf spark.sql.catalog.my_catalog=org.apache.iceberg.spark.SparkCata
 
 For more details on using S3 Acceleration, please refer to [Configuring fast, secure file transfers using Amazon S3 Transfer Acceleration](https://docs.aws.amazon.com/AmazonS3/latest/userguide/transfer-acceleration.html).
 
+### S3 Analytics Accelerator
+
+The [Analytics Accelerator Library for Amazon S3](https://github.com/awslabs/analytics-accelerator-s3) helps you accelerate access to Amazon S3 data from your applications. This open-source solution reduces processing times and compute costs for your data analytics workloads.
+
+In order to enable S3 Analytics Accelerator Library to work in Iceberg, you can set the `s3.analytics-accelerator.enabled` catalog property to `true`. By default, this property is set to `false`.
+
+For example, to use S3 Analytics Accelerator with Spark 3.5, you can start the Spark SQL shell with:
```

Comment:
nit: no need to mention Spark version

Reply:
I'm happy to remove it, just that all other sections specified it. Should we also?

Reply:
I guess we should ideally remove those all, but let’s at least remove the version for this change. This is not really 3.5 specific, and the version keeps upgrading